backend/hitas/views/utils/filters.py

```diff
@@ -22,7 +22,8 @@ class HitasFilterSet(FilterSet):
 
 class HitasCharFilter(filters.CharFilter):
     def __init__(self, *args, **kwargs):
-        kwargs["min_length"] = 3
```

Comment:
Use case here is to provide results to the user faster with less input required. e.g. as you said, in your comment below.

Reply:
but why? commit message or PR does not state that. why 2 instead of 3? why not 1?

IMO 3 is quite small already. I don't really can see a use-case where typing only two characters return any relevant results :thinking:

Reply:
No specific reason for it to be `2` other than it being less than `3`, it can also be changed to `1` 😄